README.md

```diff
@@ -71,6 +71,18 @@ UNIGE Platform is deployed with CI/CD from integral gitlab, which is synchronize
 
 **where to push?** we should make sure each repository which has both github and gitlab.astro.unige.ch versions to be synchronized between the two. Simple tooling is needed to ensure both locations are synchronous.
 
+## Github project management features: Labels, Milestones, Projects
+
+We need multi-repository management, so milestones are excluded, unfortunately.
+
+Labels are per-repository, but even if they are created independently, they can be aggregated in organization if they are called the same way, [as so](https://github.com/issues?q=is%3Aopen+is%3Aissue++archived%3Afalse+label%3Aoda-21-05+).
+
+Therefore, we can use labels to indicate given milestone, e.g. for v21.05 release: [oda-21-05](https://github.com/issues?q=is%3Aopen+is%3Aissue++archived%3Afalse+label%3Aoda-21-05+).
+
```

Comment:
I agree with the approach, but isn't this clashing a bit with the milestone concept of github? 
[https://github.com/oda-hub/dispatcher-app/milestone/1](https://github.com/oda-hub/dispatcher-app/milestone/1)

Reply:
There is some freedom about how to use different project management features of github, some can be used interchangeably. 
Most of all, milestones can be be used for multi-repo project, I specifically noted this fact in the paragraph above.

These kind of features are changing and evolving, maybe soon github will allow mutli-repo milestones, or we can even 
But for now, milestone is not good for multi-project on its own, is not shown quite as clearly as label, etc.
It is possible to use github cli, external integrations (like zenhub) etc.
But I am not sure we need this complexity as agreed policy.
That is, we can use these additional tools ourselves independently, as long as we stick to simple github-only conventions.

Reply:
I might be missing a point here, I thought milestone cannot actually be used for multi-repo, or am I confusing with something else?

Reply:
good point, I meant to say "replace" milestone or something along these lines. I modified.

Reply:
oh, I realized it is possible to mimich multi-project milestones by name. hm. I will modify, sorry for confusion!

Reply:
Oh cool, definitely has to be mentioned. Perhaps with an example, since I don't think we have ever used yet such feature as far as I remember of course.

Reply:
Well, it's in part quite recent and is evolving, so it's good to re-asses every once in a while. As long as we do not spend too much time on it. 
Please have a look now. I hope I did not miss anything.